lib/player.rb

```diff
@@ -82,6 +82,12 @@ def all_ships_placed?
     ship_count == ships_on_board
   end
 
+  def cell_fired_upon?(coord)
```

Comment:
Love that you wrote this in. I think I may be able to use this in the refactor for the fire_coordinate (which is for user input or technically auto input of firing coordinate)

Reply:
I'm all for reusing code.  And I have some ideas about our render method now that you mention refactoring.  It might be nothing, but I was wondering why the cell AND the board BOTH have render methods that reference the same exact cells.  I put down a note that I think this would be a perfect moment for a module.

Reply:
Yessss! I had the same question. I'm glad we are both thinking through our work searching for repetitive moments!